src/components/Footer/FooterNav.jsx

```diff
@@ -0,0 +1,41 @@
+import {
+  FaFacebook,
+  FaLinkedinIn,
+  FaTwitter,
+  FaWhatsapp,
+  FaYoutube
+} from 'react-icons/fa';
+
+import { FooterNavIcons } from './FooterNavIcons';
+
+export const FooterNav = () => {
+  return (
+    <>
+      <FooterNavIcons
+        elem={<FaTwitter />}
+        color="twitter"
+        href={'https://twitter.com'}
```

Comment:
why do we need all these links to socials if we are not going to have them? the product is meant to be internal so it doesn't make sense

Reply:
if we want to use exadels socials we could but i don't think it would be good until we have actual requirements from BA

Reply:
I undestand. It is like a templete, we can put there anather links. But if we do not need them what i must put inside footer?

Reply:
for now just having the component without any links is okay, we will adjust later according to BA needs anyway